css/qmwp-search-correlations.css

```diff
@@ -0,0 +1,286 @@
+.fieldsContainer {
+    font-size:0;
+    max-width: 100%;
+    position: relative;
+}
+
+.searchsubmit{
+    padding:6px 20px !important;	
+    text-transform: uppercase  !important;
+    background-color:#F16645  !important;
+    color:#FFF  !important;      
+    border-bottom-right-radius:2px  !important;
+    border-top-right-radius:2px  !important;
+    border-bottom-left-radius:0px  !important;
+    border-top-left-radius:0px  !important;
+    font-size:14px  !important;
+    height: 34px  !important;
+    border: 1px solid #EFEFEF  !important;    
+    border-left:none !important;
+    vertical-align: bottom  !important;
+    margin-bottom: 10px;
+    margin-top: 0px !important;
+}
+
+.searchsubmit:active {
+    background-color:#d65a3d  !important;
+}
+
+.selectOutputAsType{
+    border:1px solid #EFEFEF;
+    padding: 6px 10px;  
+    color: #888;
+    font-size:14px;
+    height: 34px;
+    vertical-align: bottom;
+    margin-bottom: 10px;
+    margin-top: 0px !important;
+}
+
+.selectOutputAsType:focus {
+    outline: none;
+}
+
+.searchVariable{
+    border:1px solid #EFEFEF;	
+    width:100%;
+    max-width: 400px;
+    padding: 6px 20px;       
+    border-left: 1px solid #EFEFEF;
+    color: #888;
+    font-size:14px;
+    height: 34px;
+    vertical-align: bottom;
+    margin-bottom: 10px;
+    margin-top: 0px !important;
+}
+
+[ng\:cloak], [ng-cloak], .ng-cloak {
+    display: none;
+}
+
+#searchResultRegion {     
+    margin-left:100px;
+    margin-top: 10px;
+}
+
+/* Responsive stuffs */
+@media only screen and (max-width: 768px) {
+    #searchResultRegion {     
+		margin: 15px !important;
+		overflow: hidden;
+    }
+	
+	.searchVariable {
+		width: 90%;
+	}
+	
+	a.result-title {
+		display: block;
+		width: 92%;
+	}
+	
+	a.shop-cart {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Element (a.shop-cart) is overqualified, just use .shop-cart without element name.](https://www.codacy.com/app/m_3/QuantiModo-WordPress-Plugin/file/964746528/issues/source?bid=2836188&fileBranchId=2859501#l83)